src/pages/CityPage/CityPageGuides.jsx

```diff
@@ -68,32 +65,33 @@ const CityPageGuides = () => {
     return (
       <ErrorCategories
         errorMessage={errorCategories.message}
-        filteredGuides={filteredGuides}
+        filteredGuides={guides}
         error={error}
         isLoading={isLoading}
       />
     );
   }
+
+  const cityName = city?.name || 'Unknown City';
```

Comment:
In terms of user-page interaction: the user has no way to reach a non-existent city. Since the search bar at the city search stage will show him NotFoundPage if the city does not exist. And in this case, a placeholder is displayed in case some error occurs on the backend and it cannot return the city?.name string. However, if the user manually enters something in the address bar that we do not have on the backend, then he will be given Unknown City
Unknown Country.
![image](https://github.com/user-attachments/assets/25ac6493-520d-42d2-bb78-517261421ee3)
![image](https://github.com/user-attachments/assets/625b1c5e-27cd-4862-b09b-b18b92fe791e)

Reply:
![image](https://github.com/user-attachments/assets/19a8dd4a-d865-40d5-ae05-8b205a5ed9fb)
Changed the city and country name lines. If the user himself entered something after /cities/, then we will take it and substitute it on the page, we will remove the country. And the user will be shown that there are no guides for his request.

Reply:
okay, in that case, you need to remove the line which describes the city as well